components/post_view/post_image.jsx

```diff
@@ -99,14 +103,10 @@ export default class PostImageEmbed extends React.PureComponent {
 
     render() {
         if (this.state.errored || !this.state.loaded) {
+            // scroll pop could be improved with better placeholder during !this.  state.loading
```

Comment:
just identifying this as an area that should be updated when scroll pop is addressed. Will remove if innapropriate

Reply:
I think it makes sense to make note of that, but maybe change the end to "while loading" since there's no `this.state.loading`?